content/docs/rfcs/10/README.md

```diff
@@ -455,7 +455,7 @@ Copyright and related rights waived via [CC0](https://creativecommons.org/public
 
 14. [14/WAKU2-MESSAGE spec](/spec/14)
 
-15. [14/WAKU2-RLNRELAY spec](/spec/17)
+15. [14/WAKU2-RLN-RELAY spec](/spec/17)
```

Comment:
Not part of what you changed, but I think this should be:
```suggestion
15. [17/WAKU2-RLN-RELAY spec](/spec/17)
```

Reply:
Agreed, thanks for the catch!